CellFabric/tests/test_canvas_add_wire_and_via_set.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import json
+from cell_fabric import Canvas, Wire, Via, UncoloredCenterLineGrid, EnclosureGrid
+
+def test_one():
+    c = Canvas()
+
+    m1 = c.addGen( Wire( nm='m1', layer='M1', direction='v',
+                         clg=UncoloredCenterLineGrid( width=400, pitch=720),
+                         spg=EnclosureGrid( pitch=720, stoppoint=360)))
+
+    m2 = c.addGen( Wire( nm='m2', layer='M2', direction='h',
+                         clg=UncoloredCenterLineGrid( width=400, pitch=720),
+                         spg=EnclosureGrid( pitch=720, stoppoint=360)))
+
+    v1 = c.addGen( Via( nm='v1', layer='via1', h_clg=m2.clg, v_clg=m1.clg))
+
+    for i in [0,2,4]:
+        c.addWire( m1, 'a', None, i, (0,1), (4,-1)) 
+
+    for i in [1,3,5]:
+        c.addWire( m1, 'b', None, i, (0,1), (4,-1)) 
+
+    c.addWireAndViaSet( 'a', m2, v1, 2, [0, 2, 4])
```

Comment:
@desmonddak Here is the call to addWireAndViaSet. It needs to be more complicated in implementation. The wire needs to use a stopping point grid, but the via center is on the other layers centerline grid. I need to translate to physical coordinates and then back to abstract coordinates on the stopping point grid. It works now because they line up, but that is not necessary.